clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -626,3 +779,64 @@ def correct_fieldmaps_name(to: PathLike) -> None:
 
     for z in Path(to).rglob("*phasediff_e*_ph*"):
         os.rename(z, z.parent / re.sub(r"phasediff_e[1-9]_ph", "phasediff", z.name))
+
+
+def merge_philips_diffusion(
+    json_file: PathLike,
+    number_of_parts: float,
+    run_num: str,
+) -> None:
+    """Add the dwi number in the provided JSON file for each run of Philips images.
+    The 'MultipartID' field of the input JSON file is set to 'dwi_1' or 'dwi_2' depending
+    on the run number.
+    """
+    import json
+
+    data = json.loads(json_file.read_text())
+    multipart_id = _get_multipart_id(
+        PhillipsNumberOfParts.from_int(int(number_of_parts)), run_num
+    )
+    if multipart_id is not None:
+        data["MultipartID"] = multipart_id
+        json.dump(data, json_file, indent=4)
+
+
+class PhillipsNumberOfParts(Enum):
+    """DWI scans obtained with a Phillips scanner might have
+    been divided in either 5 or 9 splits. This distinction is important
```

Comment:
For consistency's sake 

```suggestion
    been divided in either 5 or 9 parts. This distinction is important
```